script.js

```diff
@@ -0,0 +1,119 @@
+fetch('weather.json')
+    .then(response => response.json())
+    .then(data => {
+        const weatherContainer = document.getElementById('container');
+
+        //Loop through the weather data and add boxes
+        data.weather.forEach(cityWeather => {
+            const weatherBox = `
+                <div class="weather-box" data-condition="${cityWeather.condition}">
+                <div class="condition">${cityWeather.condition}</div>
+                    <img src="${cityWeather.icon}" alt="${cityWeather.condition}" class="weather-icon">
+                </div>
+            `;
+            weatherContainer.innerHTML += weatherBox;
+        });
+
+        //Load in animation
+        gsap.fromTo('.weather-box', 
+            {
+                y: -50,
+                opacity: 0
+            }, 
+            {
+                duration: 1.2, 
+                y: 0, 
+                opacity: 1, 
+                ease: "bounce.out", 
+                stagger: 0.2
+            }
+        );
+
+        //Hover over box
+        const weatherBoxes = document.querySelectorAll('.weather-box');
+        weatherBoxes.forEach(box => {
+            box.addEventListener('mouseenter', () => {
+                gsap.to(box, {
+                    duration: 0.2,
+                    y: -20,
+                    ease: "bounce.out"
+                });
+
+                //Check for condition
+                if (box.getAttribute('data-condition') === 'Clear') {
```

Comment:
This looks great, but the assignment asks to create a javascript class. Try creating a class `WeatherBox` that creates the HTML to display the box. Another hint: try creating a property in the class that add the specific gsap animation!

Reply:
Sorry about that I've changed it to include a JavaScript class that creates the weather box.